strawberryfields/api_client.py

```diff
@@ -166,6 +166,8 @@ def __init__(self, **kwargs):
 
         self.HOSTNAME = self._config["hostname"]
         self.BASE_URL = "{}://{}".format("https" if self.USE_SSL else "http", self.HOSTNAME)
+        if self._config["port"] != 443:
```

Comment:
Probably better to check against `DEFAULT_CONFIG["api"]["port"]` explicitly here